allink_core/core_apps/allink_teaser/templates/allink_teaser/includes/_image.html

```diff
@@ -1,6 +1,6 @@
 {% load allink_image_tags %}
 {% if teaser_image %}
     <div class="teaser__image">
-        {% render_image image=teaser_image alt_text=teaser_title %}
+        {% render_image image=teaser_image alt_text=teaser_title ratio="16-9" %}
```

Comment:
how is this with backwards compatibility?

Reply:
According to Nicolai this could be set to default over all past 2.6 projects